cpp/models/abm/infection.cpp

```diff
@@ -41,48 +69,72 @@ double mio::abm::Infection::get_viral_load(const TimePoint t) const
     }
 }
 
-// double mio::abm::ViralLoad::get_viral_load(const TimePoint t, const TimePoint start_date) const
-// {
-//     if (t <= m_peak / m_incline + start_date) {
-//         return m_incline * (t - start_date);
-//     }
-//     else {
-//         return m_peak + m_decline * (t - m_peak / m_incline - start_date);
-//     }
-// }
-// void mio::abm::ViralLoad::draw_viral_load()
-// {
-//     // These numbers are subject to change, They are going to be based on distributions backed from data.
-//     m_peak    = 5.0;
-//     m_incline = 1.0;
-//     m_decline = -1.0;
-// }
+double Infection::get_infectivity(const TimePoint& t) const
+{
+    return 1 / (1 + exp(-(m_log_norm_alpha + m_log_norm_beta * get_viral_load(t))));
```

Comment:
Provide reference?

Reply:
Will do.

Reply:
There already was a reference in the header. Do you want me to change it?